src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/regler/vilkår/PassBarnRegel.kt

```diff
@@ -9,12 +16,52 @@ import no.nav.tilleggsstonader.sak.vilkår.regler.SvarId
 import no.nav.tilleggsstonader.sak.vilkår.regler.Vilkårsregel
 import no.nav.tilleggsstonader.sak.vilkår.regler.jaNeiSvarRegel
 import no.nav.tilleggsstonader.sak.vilkår.regler.regelIder
+import java.time.LocalDate
+import java.util.UUID
 
 class PassBarnRegel : Vilkårsregel(
     vilkårType = VilkårType.PASS_BARN,
     regler = setOf(HAR_ALDER_LAVERE_ENN_GRENSEVERDI, UNNTAK_ALDER, DEKKES_UTGIFTER_ANNET_REGELVERK, ANNEN_FORELDER_MOTTAR_STØTTE, UTGIFTER_DOKUMENTERT),
     hovedregler = regelIder(HAR_ALDER_LAVERE_ENN_GRENSEVERDI, DEKKES_UTGIFTER_ANNET_REGELVERK, ANNEN_FORELDER_MOTTAR_STØTTE, UTGIFTER_DOKUMENTERT),
 ) {
+
+    override fun initiereDelvilkår(metadata: HovedregelMetadata, resultat: Vilkårsresultat, barnId: UUID?): List<Delvilkår> {
+        val personIdentForGjeldendeBarn = metadata.barn.firstOrNull { it.id == barnId }?.ident
+        val harFullførtFjerdetrinnSvar = if (personIdentForGjeldendeBarn == null ||
+            harFullførtFjerdetrinn(Fødselsnummer(personIdentForGjeldendeBarn).fødselsdato)
+        ) {
+            null
+        } else {
+            SvarId.NEI
+        }
```

Comment:
Kanskje man kan trekke ut alt det til 
```suggestion
        val harFullførtFjerdetrinnSvar = harFullførtFjerdetrinn(metadata, barnId)
        ...
        
        private fun harFullførtFjerdetrinn(
        metadata: HovedregelMetadata,
        barnId: UUID?
    ): SvarId? {
        val fødselsdato = metadata.barn.firstOrNull { it.id == barnId }?.let { Fødselsnummer(it.ident).fødselsdato }
        return if (fødselsdato == null || harFullførtFjerdetrinn(fødselsdato)) {
            null
        } else {
            SvarId.NEI
        }
    }
```